README.md

```diff
@@ -82,6 +82,18 @@ test-suite doctests:
 Notes
 -----
 
+* `Cabal-2.0` can be finicky about whether it builds the library component or
```

Comment:
And if possible to add, that (I don't think that) it is not possible to solve this in non-hacky way.

Some build system might build all components in greater isolation, so `cabal-doctest` test-suite cannot read `cabal_macros.h` of `library` component! I think the only solution is to add proper `cabal new-doctest` command.